pyro/advection_ppm/fluxes.py

```diff
@@ -0,0 +1,115 @@
+import numpy as np
+import pyro.mesh.patch as patch
+from pyro.mesh.reconstruction import ppm_reconstruction
+import pyro.mesh.array_indexer as ai
+
+
+def riemman(myg, u, v, ar, al, idim):
+    """Here we write the riemmann problem for the advection equation. The solution
```

Comment:
you should not need to write your own Riemann solver -- the version from advection should work fine